security/firely-auth/firely-auth-sso.rst

```diff
@@ -55,6 +55,8 @@ Multiple configuration parts are necessary to enable SSO in Firely Auth:
 
     .. Note:: 
         Note that if Firely Auth queries resources in Firely Server, it will do so via the default FHIR information model of Firely Server. Only R3 or R4 resources are supported by Firely Auth and can be used to derive the fhirUser claim this way. If the resource is not found, Firely Auth will not be able to derive the FhirUser claim. In this case, the user will not be able to log in.
+        
+        For client authentication, Firely Auth will automatically request internally a JWT access token. `Firely-Auth-Internal-FhirLookup` will be used as the client id within the token. There is no need to configure this client_id via the admin interfaces of Firely Auth.
```

Comment:
I would emphasize that this occurs during the communication with FS. Now this is not entirely clear, even though the previous part discusses the picking up of resources in FS. Would make it something like:

For setting up this connection with Firely Server, Firely Auth will automatically request a JWT access token internally, using `Firely-Auth-Internal-FhirLookup`  as the client id within the token. There is no need to configure this client_id via the admin interfaces of Firely Auth.